clients/api-rne/auth.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  HttpTooManyRequests,
+  HttpUnauthorizedError,
+} from '#clients/exceptions';
+import routes from '#clients/routes';
+import constants from '#models/constants';
+import { Information } from '#models/exceptions';
+import httpClient, { httpGet, IDefaultRequestConfig } from '#utils/network';
+import { logWarningInSentry } from '#utils/sentry';
+
+enum ECredentialType {
+  DEFAULT,
+  ACTES,
+}
+
+class RNEClient {
+  private _token = '';
+  private _currentAccountIndex = 0;
+  private accounts;
+
+  constructor(credentialType = ECredentialType.DEFAULT) {
+    this.accounts =
+      credentialType === ECredentialType.ACTES
+        ? [
+            [process.env.RNE_LOGIN_ACTES_1, process.env.RNE_PASSWORD_ACTES_1],
+            [process.env.RNE_LOGIN_ACTES_2, process.env.RNE_PASSWORD_ACTES_2],
+            [process.env.RNE_LOGIN_ACTES_3, process.env.RNE_PASSWORD_ACTES_3],
+          ]
+        : [[process.env.RNE_LOGIN, process.env.RNE_PASSWORD]];
+  }
```

Comment:
You should only use RNE_LOGIN_ACTES_X plus I think we got granted administration level account with RNE_LOGIN_ACTES_1 so I would advise to scrap the rotation account logic here.

Client will also be much simpler which is nice :)